website/source/docs/secrets/pki/index.html.md

```diff
@@ -659,3 +1183,84 @@ If you get stuck at any time, simply run `vault path-help pki` or with a subpath
     A `204` response code.
   </dd>
 </dl>
+
+### /pki/sign/
+#### POST
+
+<dl class="api">
+  <dt>Description</dt>
+  <dd>
+    Signs a new certificate based upon the provided CSR and the supplied
+    parameters, subject to the restrictions contained in the role named in the
+    endpoint. The issuing CA certificate is returned as well, so that only the
+    root CA need be in a client's trust store.
+  </dd>
+
+  <dt>Method</dt>
+  <dd>POST</dd>
+
+  <dt>URL</dt>
+  <dd>`/pki/sign/<name>`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    <ul>
+      <li>
+        <span class="param">csr</span>
+        <span class="param-flags">required</span>
+        The PEM-encoded CSR.
+      </li>
+      <li>
+        <span class="param">common_name</span>
```

Comment:
It doesn't have to be, and even if so, there's no technical reason it can't be overridden at submission time, the same as issuing any other certificates. Part of providing a nice feature set here is that you can easily generate a CSR on the command line without having to understand OpenSSL option files, then select your hostname and other values at signing time.